package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "undertaker",
-  "version": "0.13.1",
+  "version": "0.13.2",
```

Comment:
Looks like you were using an old version of the repo because it is 0.13.2 on master.  Can you fetch all changes and rebase this out?